src/json/pull_parser.cr

```diff
@@ -507,7 +507,7 @@ class JSON::PullParser
   end
 
   private def expect_kind(kind)
-    parse_exception "Expected #{kind} but was #{@kind}" unless @kind == kind
+    parse_exception "Expected #{kind} but was #{@kind} for attribute: #{@string_value}" unless @kind == kind
```

Comment:
>this result is your expected?

No, definitely not...
Thank you for pointing out the problem and showing me the better way. Let me try to fix it :)